### ISSUE TITLE
Is this compatible with laravel 7?

### ISSUE THREAD

**Issue by jblotus** [arkaitzgarro/elastic-apm-laravel#41]

**Comment by arkaitzgarro** [comment]
Hi @jblotus ,

I created a PR to add Laravel 7 compatibility. It would be great if you could test it on your side to confirm that it works.

https://github.com/arkaitzgarro/elastic-apm-laravel/pull/42

**Comment by jblotus** [comment]
awesome I will test it sometime today.

**Comment by arkaitzgarro** [comment]
> awesome I will test it sometime today.

Just for clarity, I didn't create a new release yet, you will need to tell composer to install a new version from VCS. Something like in your `composer.json` that if I'm correct:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/arkaitzgarro/elastic-apm-laravel"
        }
    ],
    "require": {
        "arkaitzgarro/elastic-apm-laravel": "dev-laravel-7-compatibity"
    }
}
```

And install it with `composer require arkaitzgarro/elastic-apm-laravel --prefer-source`.

**Comment by jblotus** [comment]
Yep thanks for the tip saved me from having to look up how to do it.

**Comment by jblotus** [comment]
This seems to have installed correctly. I did have an issue because I was relying on an earlier version of philkra/elastic-apm-agent in my composer.json - I changed that to use dev-master and it resolved.

To be honest I have not used this package before, as we have been using the philkra/elastic-apm-laravel package so I can't really report on any deeper integration. The app I'm upgrading to laravel 7 has other issues so it will take some time before I can report back any other compatibility issues.
